src/parsers/manifestjson.js

```diff
@@ -109,6 +110,19 @@ export default class ManifestJSONParser extends JSONParser {
 
     if (error.keyword === 'required') {
       baseObject = messages.MANIFEST_FIELD_REQUIRED;
+    } else if (error.keyword === 'deprecated') {
+      if (
+        this.isStaticTheme &&
+        STATIC_THEME_LWT_ALIASES.includes(error.dataPath)
+      ) {
+        baseObject = messages.MANIFEST_THEME_LWT_ALIAS;
+        // Overwrite the message with the shorter one included in the linter messages.
+        overrides.message = baseObject.message;
+      }
+      // TODO: add a messages.MANIFEST_FIELD_DEPRECATED and ensure that deprecated
```

Comment:
I'll file a follow up for this (in this PR I opted to prevent the `deprecated` keyword implementation from marking as error any other manifest property or API marked as deprecated in the schema files, because those `deprecated` schema definition have not been processed until now, and so I think that it would be better to evaluate that part separately in a follow up).